maquetado/scss/application.scss

```diff
@@ -0,0 +1,27 @@
+@import 'colors';
+@import 'equalizer';
+
+body {
```

Comment:
Maybe it is best to move this to styles.scss and then here only show the imports. Move all de body and nav styles to another file